vendor/assets/stylesheets/uswds_overrides/core/_fonts.scss

```diff
@@ -46,50 +46,50 @@
     font-url('sourcesanspro-bold-webfont.svg#source_sans_probold') format('svg');
 }
 
-@font-face {
-  font-family: 'Merriweather';
-  font-style: normal;
-  font-weight: 300;
-  src: font-url('merriweather-light-webfont.eot');
-  src: font-url('merriweather-light-webfont.eot?#iefix') format('embedded-opentype'),
-    font-url('merriweather-light-webfont.woff2') format('woff2'),
-    font-url('merriweather-light-webfont.woff') format('woff'),
-    font-url('merriweather-light-webfont.ttf') format('truetype'),
-    font-url('merriweather-light-webfont.svg#source_sans_proregular') format('svg');
-}
-
-@font-face {
-  font-family: 'Merriweather';
-  font-style: normal;
-  font-weight: 400;
-  src: font-url('merriweather-regular-webfont.eot');
-  src: font-url('merriweather-regular-webfont.eot?#iefix') format('embedded-opentype'),
-    font-url('merriweather-regular-webfont.woff2') format('woff2'),
-    font-url('merriweather-regular-webfont.woff') format('woff'),
-    font-url('merriweather-regular-webfont.ttf') format('truetype'),
-    font-url('merriweather-regular-webfont.svg#source_sans_proitalic') format('svg');
-}
-
-@font-face {
-  font-family: 'Merriweather';
-  font-style: italic;
-  font-weight: 400;
-  src: font-url('merriweather-italic-webfont.eot');
-  src: font-url('merriweather-italic-webfont.eot?#iefix') format('embedded-opentype'),
-    font-url('merriweather-italic-webfont.woff2') format('woff2'),
-    font-url('merriweather-italic-webfont.woff') format('woff'),
-    font-url('merriweather-italic-webfont.ttf') format('truetype'),
-    font-url('merriweather-italic-webfont.svg#source_sans_proitalic') format('svg');
-}
-
-@font-face {
-  font-family: 'Merriweather';
-  font-style: normal;
-  font-weight: 700;
-  src: font-url('merriweather-bold-webfont.eot');
-  src: font-url('merriweather-bold-webfont.eot?#iefix') format('embedded-opentype'),
-    font-url('merriweather-bold-webfont.woff2') format('woff2'),
-    font-url('merriweather-bold-webfont.woff') format('woff'),
-    font-url('merriweather-bold-webfont.ttf') format('truetype'),
-    font-url('merriweather-bold-webfont.svg#source_sans_probold') format('svg');
-}
+// @font-face {
```

Comment:
not sure -- the issue with keeping it is that we don't actually have these fonts in this project so the `font_url` rules are referencing files that do not exist.

The issue with *not* keeping it is that the `uswds` file that this overrides references these fonts at a different location where they do not exist.

Maybe best to just remove entirely since we are not using these font rules. Will do so now!